src/utils/utf8.c

```diff
@@ -80,6 +104,7 @@ utf8_ctx_destroy(utf8_ctx_t *ctx)
 		ucasemap_close(ctx->csm);
 	}
 	free(ctx->locale);
+	utrans_close(ctx->transliterator);
```

Comment:
- It's probably a good idea to be defensive here and just do: `if (ctx->transliterator) { ... }`
- Can you move this line before the free(), so the u* stuff is grouped together?